observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-installing.adoc

```diff
@@ -59,28 +123,6 @@ include::modules/distr-tracing-tempo-install-tempomonolithic-web-console.adoc[le
 
 include::modules/distr-tracing-tempo-install-tempomonolithic-cli.adoc[leveloffset=+2]
 
-include::modules/distr-tracing-tempo-storage-ref.adoc[leveloffset=+1]
-
-include::modules/distr-tracing-tempo-object-storage-setup-aws-sts-install.adoc[leveloffset=+2]
-
-[role="_additional-resources"]
-.Additional resources
-
-* link:https://docs.aws.amazon.com/iam/[AWS Identity and Access Management Documentation]
-* link:https://docs.aws.amazon.com/cli/[AWS Command Line Interface Documentation]
-* xref:../../../authentication/identity_providers/configuring-oidc-identity-provider.adoc#configuring-oidc-identity-provider[Configuring an OpenID Connect identity provider]
-* link:https://docs.aws.amazon.com/IAM/latest/UserGuide/reference-arns.html[Identify AWS resources with Amazon Resource Names (ARNs)]
-
-include::modules/distr-tracing-tempo-object-storage-setup-ibm-storage.adoc[leveloffset=+2]
-
-[role="_additional-resources"]
-.Additional resources
-
-* xref:../../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[Getting started with the OpenShift CLI]
-* link:https://cloud.ibm.com/docs/cli?topic=cli-getting-started[Getting started with the {ibm-cloud-title} CLI] ({ibm-cloud-title} Docs)
-* link:https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-provision[Choosing a plan and creating an instance] ({ibm-cloud-title} Docs)
-* link:https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-getting-started-cloud-object-storage&q=credential&tags=cloud-object-storage&offset=10#getting-started[Getting started with {ibm-cloud-title} Object Storage: Before you begin] ({ibm-cloud-title} Docs)
-
 [role="_additional-resources"]
 [id="additional-resources_dist-tracing-tempo-installing"]
```

Comment:
Totally out of scope and you don't have to do it, but while you're here, could you add a context to this ID as well?
```suggestion
[id="additional-resources_{context}"]
```
(In this case I'm also removing the manually typed context after the underscore, which is something you should do if you happen to apply this feedback to other IDs)

Reply:
Thank you. I reviewed ALL my assemblies and added a few fixes to this PR.